upload.py

```diff
@@ -44,19 +55,29 @@
 password = getpass.getpass()
 
 # # Open a galaxy instance using the api key.
-gi = galaxy.GalaxyInstance(url='usegalaxy.org', key=args.api_key)
+gi = galaxy.GalaxyInstance(url="https://usegalaxy.org", key=args.api_key)
 histid = gi.histories.get_most_recently_used_history()["id"]
 
 # First upload all .bw files to the galaxy FTP server.
-# NOTE: ftplib has issues with galaxy FTP, hence the lftp script.
 print(">>> Uploading .bw files to FTP server.")
-print(">>> Executing ./lftp_upload.sh %s %s %s" % (args.username, "*****", args.dir))
-p = sp.call(["./lftp_upload.sh", args.username, password, args.dir])
+
+# Set up FTPS connection
+ftps = FTPS_connect (host="usegalaxy.org", user=args.username, passwd=password)
+ftps.set_debuglevel(10)
+ftps.prot_p()
+ftps.retrlines("LIST")
```

Comment:
This just lists the contents of the directory on the FTP server. We can remove it too.